README.md

```diff
@@ -68,6 +68,35 @@ Some of the features that encourage more readable programs:
 - [ ] Trailing closure parameters can be passed outside the call parentheses.
 - [ ] Error propagation with `ErrorOr<T>` return type and dedicated `try` / `must` keywords.
 
+## Code reuse
+
+Jakt is flexible in how a project can be structured with a built-in module system.
+
+```jakt
+import a                                (1)
```

Comment:
Let's put these `(1)` in comments (`// (1)`)

Reply:
Good idea.